sql/expression/function/math_test.go

```diff
@@ -170,12 +170,12 @@ func TestCRC32(t *testing.T) {
 }
 
 func TestTrigFunctions(t *testing.T) {
-	asin := NewUnaryFunc("asin", sql.Float64, ASinFunc)
-	acos := NewUnaryFunc("acos", sql.Float64, ACosFunc)
-	atan := NewUnaryFunc("tan", sql.Float64, ATanFunc)
-	sin := NewUnaryFunc("sin", sql.Float64, SinFunc)
-	cos := NewUnaryFunc("cos", sql.Float64, CosFunc)
-	tan := NewUnaryFunc("atan", sql.Float64, TanFunc)
```

Comment:
bug fix here for correcting `NewUnaryFunc("tan", sql.Float64, ATanFunc)` and `NewUnaryFunc("atan", sql.Float64, TanFunc)`?

Reply:
yup